src/templates/PoolV2/ShareAndEarn/ShareAndEarnCard/styles.ts

```diff
@@ -0,0 +1,224 @@
+import styled, { css } from 'styled-components'
+
+export const ShareAndEarnCard = styled.div`
+  ${() => css`
+    position: relative;
+
+    /* display: flex;
+    align-items: center;
+    justify-content: space-between; */
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    gap: 3.2rem;
+    padding: 3.2rem;
+
+    border-radius: 16px;
+    border: 1px solid rgba(252, 252, 252, 0.05);
+    background: rgba(252, 252, 252, 0.05);
+
+    @media (max-width: 992px) {
+      grid-template-columns: 1fr;
+    }
+
+    &::before {
+      content: '';
+      position: absolute;
+      top: 0;
+      left: 0;
+      right: 0;
+      bottom: 0;
+
+      border-radius: 0.8rem;
+      border: 0.1rem solid transparent;
+      background: linear-gradient(270deg, #ffbf00 -1.42%, #e843c4 101.42%)
+        border-box;
+
+      -webkit-mask:
+        linear-gradient(#fff 0 0) padding-box,
+        linear-gradient(#fff 0 0);
+      -webkit-mask-composite: destination-out;
+      mask-composite: exclude;
+
+      z-index: -1;
+    }
+  `}
+`
+
+export const ShareAndEarnContainer = styled.div`
```

Comment:
Eu acho que tem que colocar Card, porque existe um `ShareAndEarn` em outro lugar provavelmente

```suggestion
export const ShareAndEarnCardContainer = styled.div`
```